### ISSUE TITLE
Do not try this command on a mac

### ISSUE THREAD

**Issue by caramelmelmel** [mubaris/yes#121]
This will max out your cpu and then you die

**Comment by 4yman-0** [comment]
Of course, It maxes out CPU on ALL PLATFORMS (Even MS Windows) if you dont use it carefully.
1. Don't just run `yes`. or you will see *horrors* like this:
```
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
y
```
2. Run it like *this*: `yes | <some-other-program>`


Edit: Use actual yes output 'y'
